contracts/main.py

```diff
@@ -0,0 +1,3 @@
+import Auction.sol
+
+print("hello")
```

Comment:
I think this is some test file you submit by accident ? shall we remove it ?

Reply:
yes! this was just a test file. feel free to remove it